wrappers/python/CMakeLists.txt

```diff
@@ -168,6 +168,12 @@ install(TARGETS pybackend2 pyrealsense2
 )
 
 target_include_directories(pybackend2 PRIVATE ../../src)
+
+else() # not BUILD_LEGACY_PYBACKEND
+
+install(TARGETS pyrealsense2
+    LIBRARY DESTINATION ${PYTHON_INSTALL_DIR}
```

Comment:
Hi @eufrizz 
I'm wondering whether all these references to `PYTHON_INSTALL_DIR` should really be to `CMAKECONFIG_PY_INSTALL_DIR`, like the other install destinations do?
Have you tried with a non-standard install location?

Reply:
No, `CMAKECONFIG_PY_INSTALL_DIR` is a different location where the cmake config files are placed, allowing consumers to use `find_package(pyrealsense2)` in their projects. The location looks something like: `/usr/local/lib/cmake/pyrealsense2/pyrealsense2Targets.cmake`.
This is not where the .so files should be installed. You'll notice a few lines above that PYTHON_INSTALL_DIR is used when built with pybackend2, but since your commit https://github.com/IntelRealSense/librealsense/commit/953bb0615112f273d1d2dfccdc4466cd4d5719f0 this is not the default, and libraries are no longer installed.

Reply:
The variable is set here in /CMake/external_pybind11.cmake: https://github.com/IntelRealSense/librealsense/blob/e1688cc318457f7dd57abcdbedd3398062db3009/CMake/external_pybind11.cmake#L69
The rsutils bindings are done in the same way: https://github.com/IntelRealSense/librealsense/blob/e1688cc318457f7dd57abcdbedd3398062db3009/third-party/rsutils/py/CMakeLists.txt#L27
As for non-standard install locations (i.e. setting CMAKE_INSTALL_PREFIX), it will ignore them and install in the Python dist-packages regardless

Reply:
I've updated it to now install ARCHIVE (I assume this is useful for building with SHARED_LIBS=false), as well as RUNTIME and EXPORT, as is done in the install a few lines above. There is already an EXPORT rule for pyrealsense2Targets below so I've left that out.